netbox_metrics_ext/metrics.py

```diff
@@ -48,12 +51,25 @@ def metric_reports():
         Iterator[GaugeMetricFamily]
             netbox_report_stats: with report name and status as labels
```

Comment:
Looks like this docstring should be updated as you're adding a third label.

Reply:
updated